crates/epaint_default_fonts/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# `epaint_default_fonts` changelog
+All notable changes to the `epaint_default_fonts` crate will be documented in this file.
+
+This file is updated upon each release.
+Changes since the last release can be found at <https://github.com/emilk/egui/compare/latest...HEAD> or by running the `scripts/generate_changelog.py` script.
```

Comment:
Then we need to also add `epaint_default_fonts` to the `crate_names` field in `scripts/generate_changelog.py`